esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -400,6 +399,11 @@ void CWsWorkunitsEx::init(IPropertyTree *cfg, const char *process, const char *s
     xpath.setf("Software/EspProcess[@name=\"%s\"]/EspService[@name=\"%s\"]/ThorSlaveLogThreadPoolSize", process, service);
     thorSlaveLogThreadPoolSize = cfg->getPropInt(xpath, THOR_SLAVE_LOG_THREAD_POOL_SIZE);
 
+    xpath.setf("Software/EspProcess[@name=\"%s\"]/EspService[@name=\"%s\"]/GetWUResultMaxSizeMB", process, service);
+    unsigned getWUResultMaxSizeMB = cfg->getPropInt(xpath);
```

Comment:
I'd prefer to drop the "Get" from both the option and the variable, I think it's misleading.

Reply:
@afishbeck - does this impact on any changes you're making to the esp configuration (that might pertain more to cloud) ?

Reply:
Sort of, It would become another item in the eclwatch as an application config.